realm/src/main/java/io/realm/internal/Table.java

```diff
@@ -320,12 +302,11 @@ public String getColumnName(long columnIndex) {
      * Returns the 0-based index of a column based on the name.
      *
      * @param columnName column name
-     * @return the index, -1 if not found
+     * @return the index, {@link #NO_MATCH} if not found
      */
     @Override
     public long getColumnIndex(String columnName) {
-        if (columnName == null)
-            throw new IllegalArgumentException("Column name can not be null.");
+        if (columnName == null) throw new IllegalArgumentException("Column name can not be null.");
```

Comment:
statements on separate lines. It's also inconsistent with the other updates to code style with { } for similar.